unit/goto-programs/restrict_function_pointers.cpp

```diff
@@ -148,3 +216,8 @@ TEST_CASE("Json conversion", "[core]")
     function_pointer_restrictions1.restrictions ==
     function_pointer_restrictions2.restrictions);
 }
+
+TEST_CASE("Get function pointer by name restrictions", "[core]")
```

Comment:
:no_entry_sign: The test should be tagged with `[goto-programs][restrict-function-pointers]`

Reply:
Done